src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, Color color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * (radius * radius);
```

Comment:
The `calculateArea` method currently returns a precise value, which is correct. However, ensure that this approach is consistent across all classes as per the previous review's suggestion to avoid rounding.